content/upcoming/enry.md

```diff
@@ -0,0 +1,181 @@
+---
+author: mcarmonaa
+date: 2017-07-03
+draft: true
+title: "enry: detecting your accent"
+image: ""
+description: "Announcing enry, a faster implementation of github/linguist in Go for programming language detection"
+categories: ["technical"]
+---
+
+## enry (⊙.☉)?
+If you don't know yet what *enry* is looking at the so descriptive name of "enry", maybe it's because you aren't the kind of person keen on films. Don't worry about it and keep reading.
+
+*[enry](https://github.com/src-d/enry)* is a tool written in Go to perform file language detection. It started as a port of *[github's project linguist](https://github.com/github/linguist)*.
+
+With all the tools that *enry*  provides,  you can know what language a file was written in, whether this file is used as documentation for a project, is a vendor file or even if it is a binary file.
+
+## Why develop a port?
+Some time ago at source{d}, we needed to detect the language of every file in every repository to perform analysing job in our previous pipeline. To face this task, we looked at [github's linguist project](https://github.com/github/linguist), a Ruby library that offered what we needed, but not addressed all our requirements though.
+
+So at this point, there was a need arising to implement an approach to fit in an environment mostly developed in Go, give a good performance and keep compatibility with its mother project. That's the reason why *enry* was born.
```

Comment:
"there was a need arising to implement" -> "a need arised to implement", I think sounds better. More: "give a good performance" -> "with good performance". More: "keep compatibility with" -> "keeping compatibility with".